src/main/java/ua/goit/timonov/enterprise/module_6_2/view/console/ConsoleInput.java

```diff
@@ -32,89 +38,99 @@
-     * @return      inputted int value
+     * @return          inputted int value
      */
     public static int inputInteger() {
-        while(true) {
-            try {
-                Scanner sc = new Scanner(System.in);
-                return sc.nextInt();
-            }
-            catch (Exception e) {
-                printLine("Wrong input! Please, input an integer value.");
-            }
-        }
+        String stringWithInt = inputSomeValue(DataType.INTEGER);
+        return Integer.valueOf(stringWithInt);
     }
 
     /**
-     * inputs int value from console, inputted value must be in defined interval including bounds
-     * @param lowBound          interval's low bound
-     * @param highBound         interval's high bound
-     * @return                  inputted int value
+     * inputs int value from console
+     * @param bounds    integer interval (low and high bound)
+     * @return          inputted int value
      */
-    public static int inputInteger(int lowBound, int highBound) {
+    public static int inputIntegerWithBounds(int... bounds) {
+        String stringWithInt = inputSomeValue(DataType.INTEGER_WITH_DIAPASON, bounds);
+        return Integer.valueOf(stringWithInt);
+    }
+
+    private static String inputSomeValue(DataType dataType, int... bounds) {
         while(true) {
+            Scanner sc = new Scanner(System.in);
+            String inputString = sc.next();
+            checkIfUserRefusesInput(inputString);
             try {
-                Scanner sc = new Scanner(System.in);
-                int value = sc.nextInt();
-                if (value < lowBound || value > highBound) {
-                    throw new IllegalArgumentException("Value should be in diapason [" + lowBound + ".." + highBound + "]");
-                }
-                return value;
+                tryToMakeParse(inputString, dataType, bounds);
+                return inputString;
             }
-            catch (IllegalArgumentException e) {
-                printLine(e.getMessage());
+            catch (ValueOutOfBoundsException e) {
+                LOGGER.info(e.getMessage());
             }
-            catch (Exception e) {
-                printLine("Wrong input! Please, input an integer value.");
+            catch (NumberFormatException e) {
+                LOGGER.info("Wrong input! Please, input proper value or input \"Q\" or \"q\" to quit");
             }
         }
     }
 
-    /**
-     * inputs float value from console
-     * @return      inputted float value
-     */
-    public static float inputFloat() {
-        while(true) {
-            try {
-                Scanner sc = new Scanner(System.in);
-                return sc.nextFloat();
-            }
-            catch (Exception e) {
-                printLine("Wrong input! Please, input a float value.");
+    private static void checkIfUserRefusesInput(String inputString) {
+        if (inputString.toUpperCase().equals(QUIT)) {
```

Comment:
invert this if to avoid NPE
Use StringUtils from Apache Commons 3 to compare ignoreing case